.github/workflows/blade-icon-to-code.yml

```diff
@@ -0,0 +1,77 @@
+name: "Blade icon to code"
+
+on:
+  workflow_dispatch:
+    inputs:
+      
+
+jobs:
+  create-pr:
+    runs-on: ubuntu-latest
+    # ensure the job can push and open a PR
+    permissions:
+      contents: write
+      pull-requests: write
+
+    steps:
+      # 1. checkout full history so we can push a new branch
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      # 2. decide what text we’ll put in the file
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '20'
+      
+      - name: Install Yarn
+        run: npm install --global yarn
+
+      - name: Install dependencies
+        run: |
+         cd packages/blade
+         yarn install --frozen-lockfile
+      
+      - name: Update icons.json
+        env:
+          FIGMA_API_TOKEN: ${{ secrets.FIGMA_API_TOKEN }}
+        run: |
+          cd packages/blade
+          node ./scripts/getSvgFromFigma.js
+  
+  
+      - name : Update icons Script
+        run: |
+         cd packages/blade
+         node ./scripts/generateIcons.mjs
+      # 4. create the pull request
+      - name : Update React Tests 
+        run: |
+         cd packages/blade
+         yarn run test:react Icons --updateSnapshot 
```

Comment:
check if it's possible to run both of these test:react test:react-native tests parallely.

Reply:
it only take around 56 sec to run these..  but will check anyway
<img width="1113" alt="Screenshot 2025-07-02 at 2 02 58 PM" src="https://github.com/user-attachments/assets/d670fe75-9671-47d7-b48a-7315f303d9be" />

Reply:
> checked i can create shards..
No don't complicate with shards.